### PR TITLE
Push changes for a github-only release.

### PR DESCRIPTION
**Pull request by AnashOommen**

### CODE REVIEW

File: src/Config/GoogleAdsConfig.cs [hunk 1]
```diff
@@ -153,24 +153,6 @@ public class GoogleAdsConfig : ConfigBase
...
-        private ConfigSetting<string> gmbLoginEmail =
-            new ConfigSetting<string>("GMBLoginEmail", "");
-
-        /// <summary>
-        /// Merchant Center ID to be used for Shopping campaigns.
-        /// </summary>
-        /// <remarks>
-        /// This setting is only for testing purposes.
-        /// </remarks>
-        private ConfigSetting<string> merchantCenterAccountId =
```
Comment (jradcliff) [inline src/Config/GoogleAdsConfig.cs:171]: Is this change intentional?

Comment (AnashOommen) [reply src/Config/GoogleAdsConfig.cs:171]: Yeah, it is. It is an internal property that I had for testing code examples, but now that FT is used, we no longer need this property. Doesn't affect users because it is marked as internal and only for testing purposes.

Comment (jradcliff) [reply src/Config/GoogleAdsConfig.cs:171]: OK, thanks for confirming.

**Review by jradcliff — APPROVED** [review]

**Review by friedenberg — APPROVED** [review]

**Review by sundquist — APPROVED** [review]
LGTM
